tmt/convert.py

```diff
@@ -116,7 +125,7 @@ def read_manual_data(testcase):
     return md_content
 
 
-def html_to_markdown(html):
+def html_to_markdown(html: str) -> Any:
```

Comment:
Isn't `md_handler.handle()` returning a string? It seems to me `str` would be better here, I'm pretty sure return value of `html_to_markdown` is never going to be `bool` or `int` or `List[Tuple[Dict[str, bool]]]`, we can make it stricter.